index.html

```diff
@@ -7,14 +7,13 @@
   </head>
 
   <body>
-
     
       <h1 id="title" style="text-align: center; text-transform: uppercase;">Microverse Survey Form</h1>
       <p style="text-align: center; text-transform: uppercase; " id="description">This Survey form is intented to collect data of all prospective students of Microverse</p>
    
       <div style="margin-top: 4%;"></div>
 
-    <form id="survey-form" action="">
+    <form id="survey-form" action="" >
       <p style="font-weight: bold; text-transform: uppercase;">Enter Your Personal Details</p>
```

Comment:
To respect best practices, Please try to use an external CSS file to style your html, using inline styling will make your code complex and hard to maintain.